whc-shared/src/test/java/com/ibm/whc/deid/shared/pojo/config/masking/DateTimeMaskingProviderConfigTest.java

```diff
@@ -18,21 +18,153 @@ public void testValidate() throws Exception {
     DateTimeMaskingProviderConfig config = new DateTimeMaskingProviderConfig();
     config.validate(null);
 
-    config.setUnexpectedInputHandling(UnexpectedMaskingInputHandler.NULL);
+    config.setFormatFixed("TTT");
+    try {
+      config.validate(null);
+      fail("expected exception");
+    } catch (InvalidMaskingConfigurationException e) {
+      System.out.println(e.getMessage());
+      assertTrue(e.getMessage().contains("`formatFixed` does not contain a valid pattern: "));
+      assertTrue(e.getMessage().contains("Unknown pattern letter: T"));
+    }
+    config.setFormatFixed("yyyy-MM-dd");
     config.validate(null);
+    config.setFormatFixed(null);
 
-    config.setFormatFixed("TTT");
+    // bad pattern
+    config.setYearDeleteNdaysOutputFormat("TXT");
     try {
       config.validate(null);
       fail("expected exception");
     } catch (InvalidMaskingConfigurationException e) {
-      assertTrue(e.getMessage().contains("`formatFixed` is not valid: "));
+      System.out.println(e.getMessage());
```

Comment:
Maybe print with System.err so it's easier to see?

Reply:
I believe our normal convention in test code is to use System.out.  I don't think this line is of particular importance to be able to find.